### ISSUE TITLE
"No supported Nvidia GPU found, use CPU instead"  on RTX 3050 Laptop

### ISSUE THREAD

**Issue by xenono1234** [RVC-Project/Retrieval-based-Voice-Conversion-WebUI#842]
It says "No supported Nvidia GPU found, use CPU instead" in windows on my RTX 3050 laptop even if I used the right command of 
`        pip install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu117
`

Using "nvcc --version" in terminal shows that I have CUDA 11.7 as well (installed myself anyway) and the PATH in environment variables is correct too. 
I am no expert in doing these things properly so might have messed somewhere! thanks in advance for the help.

**Comment by Rokkonrol** [comment]
Try this:

`pip3 install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu117 --force-reinstall `

If that doesn't work try creating a virtualenv, activating the env, then doing `pip install -r requirements.txt` followed by 
`pip3 install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu117 --force-reinstall`.

I highly recommend using Anaconda for creating a virtualenv. It's way easier and more intuitive than Windows Terminal (for me at least). If you need any more help or specific step-by-step instructions let me know.

**Comment by xenono1234** [comment]
Thank you so much for the help!

I ran the commands you mentioned but it ended giving more errors like some specific arrow version is not installed or some error with 'typing_extensions'. Since I am not very good at handling those, I decided to remove everything I extracted from RVC-beta.7z.   After that I extracted the files from RVC-beta.7z again and ran the go-web.bat and it finally detected my RTX 3050 laptop GPU! 

Also I think I should mention that I had to install platformdirs manually both the times I extracted the RVC-beta.7z
